package.json

```diff
@@ -49,10 +52,13 @@
     "eslint": "^7.3.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "expect-playwright": "^0.3.1",
```

Comment:
Is this being used?

Reply:
This is used by `jest-playwright-preset`.
I think it should not be listed as installed by the user here. I may have messed it.
I'll check.